tests/test_isPrimaryFlag.py

```diff
@@ -161,6 +161,8 @@ def setUp(self):
 
         # Characterize the image (create PSF, etc.)
         charImConfig = CharacterizeImageConfig()
+        charImConfig.measurePsf.psfDeterminer = 'piff'
+        charImConfig.measurePsf.psfDeterminer['piff'].spatialOrder = 0
```

Comment:
I could look it up, but I'm lazy. What's the default value for `spatialOrder`?

Reply:
Same as PSFEx: 2.
Swapping it to zero here just for unit test which includes very few stars.